Presentation.Web/app/models/generic/option-dto.ts

```diff
@@ -0,0 +1,9 @@
+﻿module Kitos.Models.OData.Generic {
+
+    export interface IOptionDTO<TReference extends IEntity> extends IEntity {
```

Comment:
perhaps let's put it in an odata sub folder (the module namespace is fine).. just to make it easier to find